crates/errors/src/error_codes/RT0012.md

```diff
@@ -0,0 +1,3 @@
+## RT0012
+
+Protocol violation error. This can be caused by an incompatible runtime and SDK version. If the error persists, please file a bug. 
```

Comment:
```suggestion
Protocol violation error. This can be caused by an incompatible runtime and SDK version. If the error persists, please file a [bug report](https://github.com/restatedev/restate/issues). 
```